src/misc/cache/lv_image_cache.c

```diff
@@ -59,51 +94,50 @@ void lv_image_cache_drop(const void * src)
 #endif
 }
 
-void lv_image_cache_resize(uint32_t new_size, bool evict_now)
-{
+/**********************
+ *   STATIC FUNCTIONS
+ **********************/
+
 #if LV_CACHE_DEF_SIZE > 0
```

Comment:
Should there be just one of these in this file instead of many?

Reply:
@liamHowatt Could you please explain the issue in detail? I don't understand your point.